### PR TITLE
Ft notify line manager 167733823

### PR DESCRIPTION
**Pull request by tamunoibi**
#### What does this PR do?
- Sends email notification to the user on successful creation of a trip request
- Sends a real time notifcation to the user's manager on successful creation of a trip request

#### Description of Task to be completed?
- Create email notification
- Create socket.io notification

#### How should this be manually tested?
- Create a trip
- You should receive email notification
- The users manager should get a real time notifcation

#### What are the relevant pivotal tracker stories?
- #167733823

### CODE REVIEW

File: src/controllers/profile.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  /**
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} the new user
+   * @description register a new client
+   */
+  static async getProfile(req, res) {
+    const { userId } = req.params;
+
+    const message = `<!DOCTYPE html>
```
Comment (EziamakaNV) [inline src/controllers/profile.js:15]: Hey Ib, I was wondering why you're front-end code here? As we're only dealing only with backend code

File: src/controllers/request.js [hunk 2]
```diff
@@ -31,37 +44,37 @@ export default class RequestController {
...
         departureDate,
         accommodation
       } = req.body;
 
+      const userId = req.user.id;
       const request = await Request.create({
         reason,
         type,
         passportName,
         managerId,
```
Comment (EziamakaNV) [inline src/controllers/request.js:74]: Hi Ib, I remember this morning we agreed to go with 'lineManagerId' as opposed to 'managerId'

File: src/index.js [hunk 3]
```diff
@@ -41,21 +49,26 @@ app.get('/', (req, res) => res.status(200).json({
 app.use(cors());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
-app.use(express.static(`${__dirname}/public`));
+app.use('/api/v1/trips/create', express.static(`${__dirname}/public`));
```
Comment (EziamakaNV) [inline src/index.js:52]: Hey Ib, could you please explain this line? Are you trying to serve static files if someone tries to create a trip?

File: src/models/notification.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    type: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        notEmpty: {
+          msg: 'notification type cannot be empty'
+        }
+      }
+    },
+    receiver: {
```
Comment (EziamakaNV) [inline src/models/notification.js:12]: Hi Ib, in your notification migration file, the data type listed there was string, but here it's an integer

File: src/routes/api/request.js [hunk 5]
```diff
@@ -1,11 +1,18 @@
 import { Router } from 'express';
-import { RequestController } from '../../controllers';
+import { RequestController, } from '../../controllers';
 import validateTripRequest from '../../middlewares/Validations/validateTripRequest';
 
+
 const router = Router();
 
-router.get('/', RequestController.getAllRequests);
+const foo = (io = null) => {
```
Comment (EziamakaNV) [inline src/routes/api/request.js:8]: Hi Ib, I was wondering if exporting this expression allows the router to mount properly?

File: src/index.js (hunk 3 above)
Comment (tamunoibi) [reply src/index.js:52]: No. I am creating a Html page to mimic the manager dashboard so I can show that he is receiving notifications

**Pull request closed without merging**